app/Http/Controllers/BuyersController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Buyer;
+use App\Models\Address;
+use Illuminate\Http\Request;
+
+class BuyersController extends Controller
+{
+    public function showOne($id)
+    {
+        $buyer = Buyer::query()->findOrFail($id);
+        $addresses = Address::query()->where('buyer_id', $id)->orderBy('addition_date', 'desc')->get();
```

Comment:
Адреса нужно было получить через связь между моделью покупателя и адресом:
```suggestion
        $buyer = Buyer::with(['addresses' => function ($query) {
	        $query->orderBy('created_at', 'desc');
        }])->findOrFail($id);
```